terrain_navigation_ros/config/ap_config.yaml

```diff
@@ -0,0 +1,273 @@
+# Common configuration for ArduPilot
+#
+# node:
+/mavros/**:
+  ros__parameters:
+    startup_px4_usb_quirk: false
+
+# --- system plugins ---
+
+# sys_status & sys_time connection options
+/mavros/**/conn:
+  ros__parameters:
+    heartbeat_rate: 1.0    # send heartbeat rate in Hertz
+    heartbeat_mav_type: "ONBOARD_CONTROLLER"
+    timeout: 10.0          # heartbeat timeout in seconds
```

Comment:
For guided mode, would it be safer to do something shorter since the vehicle may be close to terrain?

Reply:
Could do. This was cut and paste from mavros. I only excised the plugins that are not working correctly or not required.

Reply:
Sounds good! A more comprehensive safety analysis could be conducted. We identified other things like datum assumptions and inability to handle a moved home location.